src/tests/nativeaot/SmokeTests/SharedLibrary/SharedLibraryDriver.mjs

```diff
@@ -1,37 +1,79 @@
-import { readFile } from 'node:fs/promises';
-import { WASI } from 'wasi';
-import { argv, env } from 'node:process';
+import { Worker, parentPort, isMainThread } from 'node:worker_threads';
+import { readFile, writeFile } from 'node:fs/promises';
+import { transpile } from '@bytecodealliance/jco';
```

Comment:
I don't see the `package.json` build part of this PR, but I assume it would be necessary to run this on CI.
And it would bring JCO toolchain as binary dependency.
We already have nodejs tools as part of Browser target for Mono upstream, so this is similar.

we have `.npmrc` to use internal NPM feed. Somebody would have to push the packages there for you.
```
registry=https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public-npm/npm/registry/
```

Reply:
I added the `jco` dependency a week ago and CI seems happy about it; it seems that NPM generates a package.json automatically.  But yeah, happy to change to the internal registry if desired.

Reply:
@pavelsavara CI is currently passing.  Do you need me to switch to the internal NPM registry before we merge this PR?  If so, whom should I speak with to have the `@bytecodealliance/jco` package (and its dependencies) added to that registry?

Reply:
I done that and pushed into this branch

Reply:
There is now `https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public-npm/npm/registry/@bytecodealliance/jco/-/jco-1.3.1.tgz`

Reply:
Is this running as part of CI ? Or just manually on dev machine ?

(I can't find what's triggering this)